mlocal/scripts/find_program

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+if test -z "$1" -o -z "$2" ; then
+	self=`basename $0`
+	cat <<-EOT
+	Syntax:
+	
+	    ${self} {program name} {search path}
+	EOT
+	exit 1
+fi
+
+IFS=:
+for dir in $2 ; do
+	fn="${dir}/$1"
+	if test -x "${fn}" ; then
+		# found
+		echo "${fn}"
+		exit 0
+	fi
+done
+
+exit 1
```

Comment:
This will cause a failure in the mconfig (what I'm seeing in CircleCI) if the package is not installed rather than with the correct output when it's not found.

Bad
```
checking: cryptsetup... Exited with code 1
```

Good
```
 checking: cryptsetup... no

unable to find the cryptsetup program, need package cryptsetup-bin
```

Reply:
Fixed